.github/workflows/bin-image.yml

```diff
@@ -24,192 +24,37 @@ on:
       - 'v*'
   pull_request:
 
-env:
-  MOBYBIN_REPO_SLUG: moby/moby-bin
-  DOCKER_GITCOMMIT: ${{ github.sha }}
-  VERSION: ${{ github.ref }}
-  PLATFORM: Moby Engine - Nightly
-  PRODUCT: moby-bin
-  PACKAGER_NAME: The Moby Project
```

Comment:
We can't pass these env vars to the reusable workflow atm so keeping in draft for now

Reply:
Looks good with https://github.com/crazy-max/.github/pull/41, see https://github.com/crazy-max/.github/actions/runs/14330805496/job/40166181902#step:5:274